x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/parser/EsqlParser.java

```diff
@@ -131,6 +131,10 @@ public void exitFunctionExpression(EsqlBaseParser.FunctionExpressionContext ctx)
     private static final BaseErrorListener ERROR_LISTENER = new BaseErrorListener() {
         // replace entries that start with <comma?><space?>DEV_<space?>
         private final Pattern REPLACE_DEV = Pattern.compile(",*\\s*DEV_\\w+\\s*");
+        private final String BEFORE_REGEX = "([\\s,]*)", AFTER_REGEX = "([\\s,]|$)";
```

Comment:
I think `BEFORE_REGEX` is a bit too broad because it matches an empty string. So any future tokens ending in `lp` would match, too - like, if we introduced a `HELP` command.

Should this maybe be `(^|[\\s,]+)`?